README.rst

```diff
@@ -42,8 +42,35 @@ The created venv should be located somewhere around .tox/dev/Scripts.
 
 Versioning
 ==========
-To track the versions of this python package we use `setuptools-scm <https://pypi.org/project/setuptools-scm/>`_.
-There are different kind of ways to achieve the version tracking. We will use the `pyproject.toml` usage.
+Short background information about versioning of python packages.
```

Comment:
Soll das ein Block sein oder wolltest du das die Absätze auch wirkliche Absätze in der Preview sind?

Reply:
Sollten wirklich Absätze sein.